config/etc/system-image/writable-paths

```diff
@@ -13,6 +13,7 @@
 /tmp                                    none                    temporary   none        defaults
 /mnt                                    none                    temporary   none        defaults
 /media                                  none                    temporary   none        defaults
+/boot/androidboot                       auto                    persistent  none        none
```

Comment:
Recovery will mount system partition and get access to environment vars and to boot.img through this folder. This is indeed different to grub/uboot, where you have separated partitions to hold this kind of data. Recovery just holds a boot image, without file system.